tracks/ruby/exercises/word-count.md

```diff
@@ -0,0 +1,55 @@
+#### Intro
+Side exercise. Great to step up from basic loops and `Enumerable#each` to more powerful Enumberable methods. 
+
+#### Reasonable solutions
+```
+WORD_REGEX = /\b[\w|']+\b/
+
+  def initialize(phrase)
+    @words = phrase
+  end
+
+  def word_count
+    words.each_with_object(Hash.new(0)) do | word, count |
+      count[word] += 1
+    end
+  end
+
+  private
+
+  def words
+    @words.downcase.scan(WORD_REGEX)
+  end
+end
+
+module BookKeeping
+  VERSION = 1
+end
```

Comment:
Maybe we can remove the BookKeeping module?

Reply:
Maybe we don't need to? 😅 I was asking because I didn't see it on some other solutions, as far as I remember - but I may be missing something.

Reply:
Hmm, I think you are right though. It is a bit noisy. I just wondered how to remember that it should be in there, if it's neither in the student's code nor in the notes. 
It's easy to miss. But we need a different place for this. So, 🔥